src/dev/i18n/extractors/html.js

```diff
@@ -34,7 +34,7 @@ import { createFailError } from '../../run';
 /**
  * Find all substrings of "{{ any text }}" pattern
  */
-const ANGULAR_EXPRESSION_REGEX = /\{\{+([\s\S]*?)\}\}+/g;
+const ANGULAR_EXPRESSION_REGEX = /{{([^{}]|({([^']|('([^']|(\\'))*'))*?}))*}}+/g;
```

Comment:
__nit:__ it's growing :/ Can you please extend the comment explaining all possible cases this regex is covering?